### DISCUSSION TITLE
Avoid initial draw with clusters

### DISCUSSION THREAD

**Discussion by femanzo** [ChebanovDD/MatchSweets#11] [Q&A]
Hello!
I'm trying to start a board without any solved cluster, I could manage to do that but the animations are showing, where would you think is the best place to insert a function that solves the icons before they are shown?

**Comment by ChebanovDD** [comment]
Hello.
Actually, I didn't get what you want to achieve. Could you provide a bit more information or attach a video?

**Comment by femanzo** [comment]
Sure, for example, this is the initial game board:
![image](https://user-images.githubusercontent.com/12688686/208278664-959d9a8c-20b1-4bf9-b396-a16ceb888618.png)

You see it starts with possible matches, and I would like to remove those matches before the board is rendered and also check if there are possible moves before showing it.

**Comment by ChebanovDD** [comment]
Okay, I see.

The best way to implement this behavior is to write your own board-filling strategy by [implementing the IBoardFillStrategy\<TGridSlot\> interface](https://github.com/ChebanovDD/Match3-SDK#create-fill-strategy). And then, return a board without any solved clusters from the `GetFillJobs` method.
